pkg/kubelet/cri/streaming/portforward/httpstream.go

```diff
@@ -51,7 +51,7 @@ func handleHTTPStreams(req *http.Request, w http.ResponseWriter, portForwarder P
 	}
 	defer conn.Close()
 
-	klog.V(5).InfoS("Connection setting port forwarding streaming connection idle timeout", "connection", conn, "idleTimeout", idleTimeout)
+	klog.V(5).InfoS("Connection setting port forwarding streaming connection idle timeout", "idleTimeout", idleTimeout)
```

Comment:
Suggest moving this log information to line 46 since you have removed conn from the output ... fail to upgrade is reported as an error.. and thus no longer need two log outs..